### PR TITLE
feat:std_live_preview [ECO-506]

### PR DESCRIPTION
**Pull request by Amitkanswal**
Title- Added Live preview
Changes- updated env config, readme,license, devtools,ui/ux
Story- [ECO-506]

### CODE REVIEW

File: src/modules/contentstack/contentstack.service.ts [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
+    }
+
+    this.Stack = contentstack.Stack(this.stackConfig);
+    this.Stack.setHost("api.contentstack.io");
+    ContentstackLivePreview.init({
+      enable: true,
+      ssr: false,
+      stackSdk: this.Stack,
+      stackDetails: {
+        apiKey: "blt123445",
```
Comment (almeida-shrutika) [inline src/modules/contentstack/contentstack.service.ts:34]: @Amitkanswal why are we adding this value here?

File: .env.sample [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+CONTENTSTACK_DELIVERY_TOKEN=YOUR_DELIVERY_TOKEN
+CONTENTSTACK_ENVIRONMENT=YOUR_PUBLISHING_ENVIRONMENT
+
+# For setting region add  CONTENTSTACK_REGION=(Optional for US region) Eg- eu
+
+# For live preview 
+CONTENTSTACK_MANAGEMENT_TOKEN= 
+CONTENTSTACK_API_HOST=
+CONTENTSTACK_APP_HOST=
+CONTENTSTACK_LIVE_PREVIEW=
```
Comment (abhishek305) [inline .env.sample:15]: add value true

File: .env.sample [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
+
+# For enabling live editing tags for this project set CONTENTSTACK_LIVE_PREVIEW=true by default it is set to false
```
Comment (abhishek305) [inline .env.sample:17]: update this comment

File: src/modules/contentstack/contentstack.service.ts [hunk 4]
```diff
@@ -1,18 +1,43 @@
...
+        ssr: false,
+      },
+      stackDetails: {
+        apiKey: config.api_key,
+        environment: config.environment,
+      },
+    }
+
+    this.Stack = contentstack.Stack(this.stackConfig);
+    this.Stack.setHost("api.contentstack.io");
```
Comment (abhishek305) [inline src/modules/contentstack/contentstack.service.ts:28]: lets take this from .env file

File: src/modules/contentstack/contentstack.service.ts (hunk 1 above)
Comment (abhishek305) [inline src/modules/contentstack/contentstack.service.ts:34]: add a string "api-key" instead of 'blt123445' as SRE team may get confused

**Pull request merged**